cmd/heartbeat/heartbeat_test.go

```diff
@@ -751,6 +752,133 @@ func TestSendHeartbeats_ErrAuth_UnsetAPIKey(t *testing.T) {
 	assert.Eventually(t, func() bool { return numCalls == 0 }, time.Second, 50*time.Millisecond)
 }
 
+func TestSendHeartbeats_ErrBackoff(t *testing.T) {
```

Comment:
I can't see where backoff is being applied in this test.

Reply:
https://github.com/wakatime/wakatime-cli/pull/878/files#diff-4e09898f30bd7131a40e051bd13906a08fd019549d103d21aaa03c3e0ff5e0a6R777